subprojects/model-core/src/test/groovy/org/gradle/internal/reflect/PropertyExtractorTest.groovy

```diff
@@ -30,15 +30,16 @@ import java.lang.annotation.Retention
 import java.lang.annotation.RetentionPolicy
 import java.lang.annotation.Target
 
+@Ignore
```

Comment:
This tests still need to be converted to the new type and the test class deleted.

Reply:
Transferred every test case from here to other test classes, most of them to `DefaultTypeAnnotationMetadataStoreTest`.